extensions/amp-story/1.0/amp-story-reaction.js

```diff
@@ -419,7 +413,9 @@ export class AmpStoryReaction extends AMP.BaseElement {
       });
 
       if (this.element.hasAttribute('endpoint')) {
-        this.updateReactionData_(optionEl.optionIndex_);
+        this.executeReactionRequest_('POST', reactionValue).catch((error) => {
+          dev().error(TAG, error);
```

Comment:
If all we do is log an error, this should be handled bu the `executeReactionRequest_()` method.